mautic/mautic
# Fire event for when plugin integration form is saved

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | n
| New feature?  | y 
| BC breaks?    | n
| Deprecations? | n
| Fixed issues  |  

## Description

This dispatches an event when a plugin integration config is saved to allow listeners to execute tasks accordingly.

It also fixes a UI issue with the config form when no features are enabled, only requires keys when the integration is published, and prevents a couple PHP notices found while testing this.

## Steps to test this PR

This is mainly developer oriented so apply the PR, delete cache and ensure a integration's config can still be saved.
